chapter_23_debugging_prod.asciidoc

```diff
@@ -464,6 +464,8 @@ FAILED (errors=1)
 
 It fails because our test utility function `create_pre_authenticated_session()`
 only acts on the local database.
+// DAVID: This feels like a logical leap. As a reader, I'm unsure whether I was supposed
+// to have known this, or if you're just saying 'trust me, I'll save you some time by telling you why'.
```

Comment:
thanks for this and all the comments david, really good to have this perspective.  will fix!